### ISSUE TITLE
Terminal resizing

### ISSUE THREAD

**Issue by mesudip** [antonmedv/countdown#14]
It doesn't handle resizing the terminal window, It accommodates the terminal size in the beginning and then doesn't care about the window resizing.

P.S. I suggest  starting the timer from zero and increasing when the countdown is started without any argument.

**Comment by antonmedv** [comment]
> P.S. I suggest starting the timer from zero and increasing when the countdown is started without any argument.

But this isn't countdown. Another tool is needed! 😄

**Comment by nyngwang** [comment]
@mesudip You can turn your screen upside down then it will countup.

**Comment by Timgrau** [comment]
@nyngwang sometimes usable, but then its a stopwatch :thinking: 
P.S. @mesudip Terminal resizing should be implemented by pressing `r` or `R`, created a PR for that

**Comment by nyngwang** [comment]
> @nyngwang sometimes usable, but then its a stopwatch 🤔

Yup for example sometimes I need to know how much time I have spent on presentation slides.

**Comment by sneak** [comment]
+1

**Comment by sfmqrb** [comment]
I think this issue is forgotten.
Resizing terminal window is an important feature to me. 
I also see a related pr #18. I didn't check it myself though. Is there any reason not merging it?

**Comment by sfmqrb** [comment]
@antonmedv
